### PR TITLE
Initial WIP implementation of Consul transport

### PR DESCRIPTION
**Pull request by deejay1**

### CODE REVIEW

File: alle_requests/adapters/service.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                netloc = '{}@{}:{}'.format(url.username,
+                                           instance['ServiceAddress'],
+                                           instance['ServicePort'])
+            else:
+                netloc = '{}:{}@{}:{}'.format(url.username, url.password,
+                                              instance['ServiceAddress'],
+                                              instance['ServicePort'])
+        else:
+            netloc = '{}:{}'.format(instance['ServiceAddress'],
+                                    instance['ServicePort'])
```
Comment (h00dy) [inline alle_requests/adapters/service.py]: Mabe something like that:

if url.username and url.password:
   netloc = ...
elif url.username:
   netloc = ...
else:
   netloc = ....

Comment (deejay1) [inline alle_requests/adapters/service.py]: yeah, should be easier to read, thx

**Comment by ch40s3k** [comment]
:+1:

**Comment by h00dy** [comment]
:+1:

**Comment by pnuckowski** [comment]
:+1:

**Pull request merged**